kalite/topic_tools/__init__.py

```diff
@@ -168,12 +187,18 @@ def get_exercise_cache(force=False, language=None):
         EXERCISES = {}
     if EXERCISES.get(language) is None:
         if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
-            exercises = softload_json(settings.EXERCISES_FILEPATH + "_" + language + ".cache", logger=logging.debug, raises=False)
+            exercises = softload_json(
+                cache_file_path("exercises_{0}.json".format(language)),
+                logger=logging.debug,
+                raises=False
+            )
             if exercises:
                 EXERCISES[language] = exercises
                 return EXERCISES[language]
         EXERCISES[language] = softload_json(settings.EXERCISES_FILEPATH, logger=logging.debug, raises=False)
-        if language == "en":  # English-language exercises live in application space, translations in user space
+        
+        # English-language exercises live in application space, translations in user space
+        if language == "en":
```

Comment:
Yes it just made the line exceed 80 chars, guessing it was a fly-by subliminal action :)